app/store.js

```diff
@@ -18,16 +18,35 @@ module.exports = ({
                     })
             },
 
+
             findAllUsers: () => {
                 return User.find()
-                    .then((responce) => {
+                    .then((response) => {
                         console.log("Listing all users now...");
-                        return responce;
+                        return response;
                     })
-            }
+            },
 
 
+            findSingleUser: (req) => {
```

Comment:
again not necessary to wrap but if you want to wrap it as you did put userId in the params instead of request...

Reply:
Applied this to updateUser, findUser & deleteUserById